proposals/nnnn-structured-concurrency.md

```diff
@@ -377,13 +377,43 @@ Much of the proposed implementation of structured concurrency is in the APIs for
 
 #### `Task` type
 
-The `Task` type is an empty `enum` that is used as a namespace for types and operations related to task management.
+The `Task` type is used as a namespace for types and operations related to task management. 
+
+```swift
+struct Task: Equatable, Hashable { ... }
+```
+
+The "current task" can be obtained from inside of an async function using the `current` static function, like this:
+
+```swift
+func foo() async { 
+  let task = await Task.current()
```

Comment:
if Kavon's async properties would land this could be `await Task.current` -- more consistent with all the other APIs then

Reply:
fyi @kavon